go/sql/user_channel.go

```diff
@@ -0,0 +1,32 @@
+package sql
```

Comment:
Is this file generated or written by hand?

Reply:
It's by hand.